src/main/java/AlexLink/HomeWork/String/TaskAdd.java

```diff
@@ -0,0 +1,33 @@
+package AlexLink.HomeWork.String;
+
+//Дополнительное задание
+//
+//1) Найти в строке не только запятые, но и другие знаки препинания.
+//Подсчитать общее их количество.
+
+
+import java.util.Arrays;
+
+public class TaskAdd {
+    public static void main(String[] args) {
+        final String SENTENCE = "Ехал я по городу и услышал топот! Обернулся, а там стая лошадей. " +
```

Comment:
we use uppercase for static final variable. see Code Convension

Reply:
fixed it